### PR TITLE
Update privacy policy and terms of service for enterprise

### PR DESCRIPTION
**Pull request by ramsays**
closes gradebook/engineering#26

Added verbage related to enterprise contracts with educational institutions.

### CODE REVIEW

File: privacy.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ Absolutely nothing.
 
 Our ads keep Gradebook free for you to use. We work directly with local advertisers to ensure relevant advertising for you. These ads are randomly shown and no personal or activity information is used to determine which ads to show.
 
+## How does my school use my data?
+
+If your school is a Gradebook partner, they will be able to see anonymized statistics for your school. Your schools' staff (e.g. academic advisors) can request access to see your grades, but you will have to explicitly grant it.
```
Comment (vikaspotluri123) [inline privacy.md]:
```suggestion
If your school or organization is a Gradebook partner, they will be able to see anonymized statistics for your school. Your schools' staff (e.g. academic advisors) can request access to view your account for a limited time period, which you will have to approve.
```

File: tos.md [hunk 2]
```diff
@@ -10,11 +10,13 @@ Details are provided below, but in general they:
 
 ## Affiliation
 
-Gradebook is not affiliated with, endorsed, or funded by any educational institution or government entity. As such, FERPA regulations do not apply to any data entered on this site. Additionally, by entering data into Gradebook, users agree that these data are NOT official education records of the user or another party. Gradebook should solely be used as a “What-If” analysis tool and should never be used as an official record of educational information.
+Gradebook is generally not affiliated with, endorsed, or funded by any educational institution or government entity. Depending on your school, your school's Gradebook may be affiliated with your educational institution or organization. In such cases, it will be clearly noted that Gradebook has partnered with your school.
```
Comment (vikaspotluri123) [inline tos.md]:
```suggestion
Your Gradebook account is always associated with a school or organization. To remain free, Gradebook has partnerships with some of these organizations. We will generally denote if your account is linked to a partnership, but if you have any concerns, you can always contact us.
```

It's still a bit wordy

File: tos.md [hunk 3]
```diff
@@ -10,11 +10,13 @@ Details are provided below, but in general they:
...
+
+Gradebook does not serve as an offical academic record. As such, FERPA regulations do not apply to any data entered on this site. Additionally, by entering data into Gradebook, you agree that these data are NOT official education records of the user or another party. Gradebook should solely be used as a “What-If” analysis tool and should never be used as an official record of educational information.
```
Comment (vikaspotluri123) [inline tos.md]:
```suggestion
Gradebook does not serve as an offical academic record. As such, data submitted to Gradebook is not bound by FERPA regulations. Additionally, by entering data into Gradebook, you agree that these data are NOT official education records of the user or another party. Gradebook should solely be used as a “What-If” analysis tool, and should never be used as an official record of educational information.
```

File: tos.md (hunk 2 above)
Comment (ramsays) [reply tos.md]: It works, gets the point across.

**Review by vikaspotluri123 — APPROVED** [review]

File: tos.md [hunk 4]
```diff
@@ -10,11 +10,13 @@ Details are provided below, but in general they:
 
 ## Affiliation
 
-Gradebook is not affiliated with, endorsed, or funded by any educational institution or government entity. As such, FERPA regulations do not apply to any data entered on this site. Additionally, by entering data into Gradebook, users agree that these data are NOT official education records of the user or another party. Gradebook should solely be used as a “What-If” analysis tool and should never be used as an official record of educational information.
+Your Gradebook account is always associated with a school or organization. To remain free, Gradebook has partnerships with some of these organizations. We will generally denote if your account is linked to a partnership, but if you have any concerns, you can always contact us.
```
Comment (joshcosta) [inline tos.md:13]:
```suggestion
Your Gradebook account may be associated with a school or organization. To remain free, Gradebook has partnerships with some of these organizations. We will generally denote if your account is linked to a partnership, but if you have any concerns, you can always contact us or read our <a href="/privacy-policy">Privacy Policy</a> for more details.
```

**Review by joshcosta — APPROVED** [review]
My only suggestion accounts for non-affiliated accounts on the.gradebook.app and links to our privacy policy from the affiliation section of the terms of service.

**Review by joshcosta — CHANGES_REQUESTED** [review]
see above

File: tos.md (hunk 4 above)
Comment (vikaspotluri123) [reply tos.md:13]: The point I was trying to get across is:

When we sign a user up, they pick associate themselves with a school or organization, or they're associated with the "generic" organization.

Comment (ramsays) [reply tos.md:13]: I think I will leave it as is. An account is technically always associated with a school, plus linking to the Privacy Policy doesn't seem to make much sense here.

**Review by joshcosta — APPROVED** [review]
